openstack/standalone-cinder/pkg/provisioner/provisioner.go

```diff
@@ -36,7 +36,10 @@ const (
 	ProvisionerIDAnn = "standaloneCinderProvisionerIdentity"
 
 	// CinderVolumeID is an annotation to store the ID of the associated cinder volume
-	CinderVolumeID = "cinderVolumeId"
+	CinderVolumeID = "cinderVolumeID"
+
+	// ShouldProvisionVolumeAnn is an annotation to idetify whether to create new cinder volume or using existing volume
+	ShouldProvisionVolumeAnn = "pv.kubernetes.io/cinderVolumeID"
```

Comment:
How would you support provisioning via clone in the future?  Maybe add another annotation: "pv.kubernetes.io/clone" ?

Reply:
maybe I can rename this ShouldProvisionVolumeAnn annotation, could we have an annotation list for the future and arrange their name together.